pages/osx/networkquality.md

```diff
@@ -0,0 +1,19 @@
+# networkQuality
+
+> Measure the network quality by connecting to the Internet.
```

Comment:
I think we don't have a better link than this right now.
```suggestion
> Measure the network quality by connecting to the internet.
> More information: <https://support.apple.com/HT212313>.
```

Reply:
Yep, I tried searching for the page -- could not find anything :(